Forms/Access SVN - Options.ACF

```diff
@@ -1088,1121 +1132,10 @@ Begin Form
                     WebImagePaddingRight =1
                     WebImagePaddingBottom =1
                 End
-                Begin Label

```

Comment:
Please put an image back - this? https://favpng.com/png_view/fork-branching-source-code-version-control-fork-png/mce0WrLp